### ISSUE TITLE
roseSpice.png causing error on game load.

### ISSUE THREAD

**Issue by Undeadbanana** [pether-pg/ONI_Mods_byPether#59]
This is the error code

`Error in GameAssets(Clone).Assets.OnPrefabInit 
System.IO.IOException: Win32 IO returned 362. Path: C:\Users\undea\OneDrive\Documents\Klei\OxygenNotIncluded\mods\Steam\2924528762\Sprites\roseSpice_125.png
  at System.IO.FileStream.ReadData (System.Runtime.InteropServices.SafeHandle safeHandle, System.Byte[] buf, System.Int32 offset, System.Int32 count) [0x0002d] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.IO.FileStream.ReadInternal (System.Byte[] dest, System.Int32 offset, System.Int32 count) [0x00026] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.IO.FileStream.Read (System.Byte[] array, System.Int32 offset, System.Int32 count) [0x000a1] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.IO.File.ReadAllBytes (System.String path) [0x0002f] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at FragrantFlowers.FragrantFlowers_Patches_AllCommon+Assets_OnPrefabInit_Patch.LoadAndAddSprite (System.String spriteName) [0x00035] in <5747216dd55949c09886421bdafeb593>:0 
  at FragrantFlowers.FragrantFlowers_Patches_AllCommon+Assets_OnPrefabInit_Patch.Postfix () [0x00000] in <5747216dd55949c09886421bdafeb593>:0 
  at (wrapper dynamic-method) Assets.Assets.OnPrefabInit_Patch5(Assets)
  at KMonoBehaviour.InitializeComponent () [0x00068] in <0f4b778ef79c497e89f6ee18303840cf>:0 

  at UnityEngine.Debug.LogError (System.Object message, UnityEngine.Object context) [0x00000] in <72b60a3dd8cd4f12a155b761a1af9144>:0 
  at Debug.LogError (System.Object obj, UnityEngine.Object context) [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at DebugUtil.LogErrorArgs (UnityEngine.Object context, System.Object[] objs) [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at DebugUtil.LogException (UnityEngine.Object context, System.String errorMessage, System.Exception e) [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at KMonoBehaviour.InitializeComponent () [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at KMonoBehaviour.Awake () [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at UnityEngine.Object.Internal_InstantiateSingleWithParent_Injected (UnityEngine.Object data, UnityEngine.Transform parent, UnityEngine.Vector3& pos, UnityEngine.Quaternion& rot) [0x00000] in <72b60a3dd8cd4f12a155b761a1af9144>:0 
  at UnityEngine.Object.Internal_InstantiateSingleWithParent (UnityEngine.Object data, UnityEngine.Transform parent, UnityEngine.Vector3 pos, UnityEngine.Quaternion rot) [0x00000] in <72b60a3dd8cd4f12a155b761a1af9144>:0 
  at UnityEngine.Object.Instantiate (UnityEngine.Object original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.Transform parent) [0x00000] in <72b60a3dd8cd4f12a155b761a1af9144>:0 
  at UnityEngine.Object.Instantiate[T] (T original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.Transform parent) [0x00000] in <72b60a3dd8cd4f12a155b761a1af9144>:0 
  at Util.KInstantiate (UnityEngine.GameObject original, UnityEngine.Vector3 position, UnityEngine.Quaternion rotation, UnityEngine.GameObject parent, System.String name, System.Boolean initialize_id, System.Int32 gameLayer) [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at Util.KInstantiate (UnityEngine.GameObject original, UnityEngine.GameObject parent, System.String name) [0x00000] in <0f4b778ef79c497e89f6ee18303840cf>:0 
  at LaunchInitializer.Update () [0x00000] in <cd4b9bd5aa6c4ec38ec00dca1dc79105>:0 
Build: U47-562984-S`

Placed mod at top of list and still gives error.

**Comment by Undeadbanana** [comment]
Just realized I started the games first load while OneDrive was loaded, when it's enabled I've gotten issues with mod downloads and that error code seems to be indicating a missing asset?. I'm going to try and delete mod folder and redownload.

**Comment by pether-pg** [comment]
it looks like the game had some issues loading the file from your memory... strange.
there is nothing I can do to fix your memory, but I can do some work around in case that happened so it didn't crash your game. due to IRL things it may take a little longer, hopefully no more than a week...

Thanks for your report!

**Comment by Undeadbanana** [comment]
Closing this. It was OneDrive's fault; I had just reinstalled, loaded most other applications then started downloading games, ONI is weird though. 

Because everything was still in the cloud, during game load the games tries to load all the files and gets interrupted during some processes (like downloading a mod) and there were partial downloads. 

After several hours of trying to force a redownload, it finally downloaded on its own after I had given up. Now after everything's properly configured, the mod loads. Thanks for making some great mods.

**Comment by Undeadbanana** [comment]
Closing this. It was OneDrive's fault; I had just reinstalled, loaded most other applications then started downloading games, ONI is weird though. 

Because everything was still in the cloud, during game load the games tries to load all the files and gets interrupted during some processes (like downloading a mod) and there were partial downloads. 

After several hours of trying to force a redownload, it finally downloaded on its own after I had given up. Now after everythings properly configured, the mod loads. Thanks for making some great mods.
